src/steamship/data/file.py

```diff
@@ -23,15 +22,9 @@
 
 
 class FileUploadType(str, Enum):
-    FILE = "file"  # The CreateRequest contains a file upload that should be used
-    VALUE = "value"  # The Create Request contains a `text` field that should be used
```

Comment:
Flagging to make sure removing VALUE was intentional. I think it was and that that's correct, but just being safe :)